week-3/22-checkout/index.html

```diff
@@ -37,8 +37,12 @@ <h1 class="shopping-basket__title">Your Shopping Basket</h1>
         <div class="checkout">
           <h2 class="checkout__title">Checkout</h2>
           <form method="GET" id="checkout-form" class="checkout__form">
-            <!-- Add the form fields here -->
-
+<label> for="customer-name">customer-name</label>
```

Comment:
The text values for the labels should be normal text the user will see on screen, so something like "Name" rather than "customer-name", and "Email" rather than "customer-email".